app/data_models/questionnaire_store.py

```diff
@@ -77,8 +80,12 @@ def save(self):
         collection_exercise_sid = (
             self.collection_exercise_sid or self._metadata["collection_exercise_sid"]
         )
+        response_expires_at = self._metadata.get("response_expires_at")
         self._storage.save(
             data=data,
             collection_exercise_sid=collection_exercise_sid,
             submitted_at=self.submitted_at,
+            expires_at=dateutil.parser.parse(response_expires_at)
```

Comment:
This should be consistent with `metadata_parser.py`, which uses `datetime.strptime()`. I prefer this dateutil.

Reply:
True, and I would prefer dateutil usually too, but in this case I think I'll use strptime to make sure we still have a properly formatted ISO timestamp rather than allowing dateutil to parse any valid timestamp

Reply:
Ya, fair enough, but it wouldn't ever get to this if it wasn't ISO 8601. But happy with either.